app/scripts/modules/core/delivery/executionGroup/executionGroup.less

```diff
@@ -31,7 +31,6 @@
     color: #ffffff;
     background-color: #777;
     text-transform: uppercase;
-    min-width: 72px;
```

Comment:
I don't recall exactly why we did this, but I am pretty sure it was for the wrong reasons during the redesign, and results in labels like "TEST" being slightly off-center, which looks dumb.